packages/world/ts/node/render-solidity/renderSystemLibrary.ts

```diff
@@ -312,3 +314,16 @@ function renderReturnParameters(returnParameters: string[]) {
 
   return `returns (${renderArguments(returnParameters)})`;
 }
+
+function formatParams(params: string[]) {
+  // Use auxiliary argument names for arguments without names
```

Comment:
ooc do they need names?

Reply:
yes, you need them to encode the system call. The other option was to use the default value for each type, imo that is not worth it (we would need to parse and figure out the underlying type of user types, define empty structs, etc)

Reply:
added more context in the PR description:
> This is needed so that systems can have functions with arguments that don't have a name (useful if you are making the system inherit from an interface and are not using all args). In the current implementation you would get a compile error due to the encoding of the system calls.